docs/devguide/docs/swagger-docs.yaml

```diff
@@ -312,13 +312,39 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/error_response'
-      requestBody:
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/benchmark_request'
-        description: The bench mark to add
-        required: true
+    get:
```

Comment:
need to merge it again
some of swagger deleted

Reply:
fixed